### PR TITLE
fix custom build + displaying name

### PR DESCRIPTION
**Pull request by tooomm**
## Related Ticket(s)
- Fixes https://github.com/Cockatrice/Cockatrice/pull/2570#discussion_r114050968

## Short roundup of the initial problem
Introduced in https://github.com/Cockatrice/Cockatrice/pull/2570/commits/b6ecbcc32633f6d1bd19ca5aaba5edb9095892d1
- custom builds had the `<br>` tag in their name
- html code in translation string on transifex

## What will change with this Pull Request?
- removed `<br>` tags
The styling is imo also better without them

### CODE REVIEW

**Comment by tooomm** [comment]
If we think that `Version 2.3.17-beta004 (2017-04-27)` is too large in one line we can use the suggested way of github: `v2.3.17-beta004 (2017-04-27)` or drop "Version" completely!

By the way, it was annoying to need to fix the linebreaks when writing an issue because the copy and paste from `About` also had them. ;)

File: cmake/getversion.cmake [hunk 1]
```diff
@@ -201,7 +201,7 @@ if(PROJECT_VERSION_LABEL)
 	set(PROJECT_VERSION "${PROJECT_VERSION}-${PROJECT_VERSION_LABEL}")
 endif()
 
-set(PROJECT_VERSION_FRIENDLY "${PROJECT_VERSION}<br>(${GIT_COMMIT_DATE_FRIENDLY})")
+set(PROJECT_VERSION_FRIENDLY "${PROJECT_VERSION}(${GIT_COMMIT_DATE_FRIENDLY})")
```
Comment (ctrlaltca) [inline cmake/getversion.cmake]: There should be a space here between the version and the date

File: cockatrice/src/window_main.cpp [hunk 2]
```diff
@@ -285,7 +285,7 @@ void MainWindow::actAbout()
 {
     QMessageBox mb(QMessageBox::NoIcon, tr("About Cockatrice"), QString(
         "<font size=\"8\"><b>Cockatrice</b></font><br>"
-        + tr("Version %1").arg(VERSION_STRING)
+        + tr("Version") + QString(" %1").arg(VERSION_STRING)
```
Comment (tooomm) [inline cockatrice/src/window_main.cpp]: Here I can use `tr("Version") + VERSION_STRING` instead?

Comment (ctrlaltca) [inline cockatrice/src/window_main.cpp]: That would miss a space and require an implicit cast from a const char* to QString.

Comment (tooomm) [inline cockatrice/src/window_main.cpp]: Ok, so it's better the way I did it right now.

**Pull request merged**